.github/workflows/continuous-integration.yml

```diff
@@ -256,6 +257,12 @@ jobs:
           - php-version: "8.4"
             postgres-version: "16"
             extension: "pdo_pgsql"
+          - php-version: "8.4"
+            postgres-version: "17"
+            extension: "pgsql"
+          - php-version: "8.4"
+            postgres-version: "17"
+            extension: "pdo_pgsql"
```

Comment:
Let's not add more jobs to the CI matrix. It's already too big. All the jobs in the `include` list are supposed to test against the latest Postgres version, so let's instead just replace `16` with `17` in that list.